packages/@okta/vuepress-site/docs/reference/api/domains/index.md

```diff
@@ -0,0 +1,671 @@
+---
+title: Domains
+category: management
+---
+
+# Domains API
+
+<ApiLifecycle access="ea" />
+
+The Okta Domains API provides operations to manage custom domains for your organization.
+
+## Getting Started
+
+Explore the Domains API: [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/6540fce00064f4f440d3)
```

Comment:
Can you please (1) update this collection to use the v2.1 format and (2) upload the JSON file for this collection to `okta-developer-docs/packages/@okta/vuepress-site/.vuepress/public/docs/api/postman/`

Reply:
added ` /public/docs/api/postman/domains.json` file in v2.1 format